editor/plugins/gizmos/mesh_instance_3d_custom_aabb_gizmo_plugin.cpp

```diff
@@ -0,0 +1,78 @@
+/**************************************************************************/
+/*  mesh_instance_3d_custom_aabb_gizmo_plugin.cpp                         */
```

Comment:
I'd rename the class to GeometryInstance3DGizmoPlugin, as it's used on any node that inherits from VisualInstance3D.